RelVal/testbed/resubmitTaskChain.py

```diff
@@ -1,199 +0,0 @@
-#!/usr/bin/env python
-
-import os
-import sys
-import urllib
-import httplib
-import re
-import json
-import pprint
-from copy import deepcopy
-from WMCore.WMSpec.WMWorkload import WMWorkloadHelper
-
-#url = "cmsweb-testbed.cern.ch"
-url = "cmsweb.cern.ch"
-reqmgrCouchURL = "https://"+url+"/couchdb/reqmgr_workload_cache"
-
-def approveRequest(url,workflow):
-    params = {"requestName": workflow,
-              "status": "assignment-approved"}
-
-    encodedParams = urllib.urlencode(params)
-    headers  =  {"Content-type": "application/x-www-form-urlencoded",
-                 "Accept": "text/plain"}
-
-    conn  =  httplib.HTTPSConnection(url, cert_file = os.getenv('X509_USER_PROXY'), key_file = os.getenv('X509_USER_PROXY'))
-    #conn  =  httplib.HTTPConnection(url)
-    conn.request("PUT",  "/reqmgr/reqMgr/request", encodedParams, headers)
-    response = conn.getresponse()
-    if response.status != 200:
-        print 'could not approve request with following parameters:'
-        for item in params.keys():
-            print item + ": " + str(params[item])
-        print 'Response from http call:'
-        print 'Status:',response.status,'Reason:',response.reason
-        print 'Explanation:'
-        data = response.read()
-        print data
-        print "Exiting!"
-        sys.exit(1)
-    conn.close()
-    print 'Cloned workflow:',workflow
-    return
-
-
-
-def retrieveSchema(workflowName):
-    specURL = os.path.join(reqmgrCouchURL, workflowName, "spec")
-    helper = WMWorkloadHelper()
-    helper.load(specURL)
-    schema = {}
-    for (key, value) in helper.data.request.schema.dictionary_whole_tree_().iteritems():
-        if key == 'ProdConfigCacheID':
-            schema['ConfigCacheID'] = value
-        elif key=='ProcConfigCacheID':
-            schema['ConfigCacheID'] = value
-        elif key=='RequestSizeEvents':
-            schema['RequestNumEvents'] = value
-        elif key=='ProcessingString' and value == {}:
-            continue
-        elif key=='AcquisitionEra' and value == {}:
-            continue
-        elif key=='SkimConfigs' and not value:
-            continue
-        elif key=='SiteWhitelist' and not value:
-            continue
-        elif value != None:
-            schema[key] = value
-#    print "Retrieved schema:"
-#    pprint.pprint(schema)
-#    sys.exit(1)
-    request = deepcopy(schema)
-
-    ### Now changing the parameters according to HG1309
-    x = 1
-    while x <= schema['TaskChain']:
```

Comment:
Nope, this one! Which is not really important, but helpful when we need to clone an old workflow with old argument names.